### DISCUSSION TITLE
Issue edit content not the same as current version

### DISCUSSION THREAD

**Discussion by fish-404** [community/community#39616] [Projects and Issues]
### Select Topic Area

Bug

### Body

I use an issue to record something. When I edit the issue's description, it's sometimes missing content from my last edition. I keep the window opened for a long time, when I click the edit button, it returned to a very old history version edition. I can get the history edited comperation in the edited history. It happens not only once. 

I wonder know which case can lead to this case, and how could I avoid, is it possible to roll back to the history edition ? Thank you. 

This is the [issue]( https://github.com/fish-404/fish-404.github.io/issues/11) I meet this problem, you can see the edited history to compare. I never delete all the text, but the edit buffer does for me, and when I only want to edit the small place, I didn't notice that and click update. 

One more strange behavior may relate to this problem is, when I have nothing drafted in my issue and move the another tab in my browser, the issue tab will turn to **Draft** hint. When I move to issue tab and click edit and update again (update nothing, but it returns to last version), the **Draft** hint disappears. I guess the "Update" action not be sent correctly but the textarea content (or we can say the HTML parsed content has changed, it doesn't base on which content is committed successfully, but base on the user committed).

Firefox Developer Edition 107.0b6(64-bit)  in Windows 11 22000.1219

**Comment by ossanna16** [comment]
Hi there @fish-404 and welcome to our community! Thank you for reporting this bug!

**Comment by jcfr** [comment]
_To follow up on this, while creating a similar report, I realized this discussion existed. You will find below, the report I was on my way to submit before searching for similar discussions._

--------------------------------
**Title:** Editing issue with check list may be out-of-sync after exiting Edit mode

## Summary

Despite of having both the rendered view and the edit history associated the issue reflecting the edit, clicking on edit doesn't display the expected content.

## Details

For example, in the context of issue https://github.com/Slicer/Slicer/issues/6652, we can see that the rendered markdown as well as the edit history correctly depict the change, but the editor doesn't show the expected content.

Trying to edit the issue from another workstation of from an _incognito_ tab display the expected content in the issue editor.

| Rendered markdown | Edit history | Editor content |
|---|---|---|
| ![image](https://user-images.githubusercontent.com/219043/203655268-12a99307-63e4-467b-9883-8f9f8345a612.png) | ![image](https://user-images.githubusercontent.com/219043/203655336-f2c2c114-6552-45d9-919b-538f99384e69.png) | ![image](https://user-images.githubusercontent.com/219043/203655418-6ebe6a04-46b6-4833-bc43-3c7bf306f031.png) |

## Steps to reproduce

1. Create an issue with the following content

   ```
   * [ ] Item 1
   * [ ] Item 2
   * [ ] Item 3
   ```

2. Check the box associated with `Item 1`

3. Click edit to verify the box is checked. It is checked :heavy_check_mark:

4. Click `Cancel`, it displays the "Are you sure you want to discard your unsaved change?"

   * Since **no changes** where done, click `OK`

5. Click again on `Edit` ... and you can observe the last check box is not checked.

The issue can be reproduced in an other repository. For example, https://github.com/jcfr/sandbox/issues/8 and https://github.com/jcfr/sandbox/issues/7

--------------------------------

**Reply by fish-404** [reply]
Hi, will you reproduce this every time? For me, I sometimes get the strange loss like yours, but sometimes it save successfully. I wonder if network or other possible reasons to cause this problem.
